static/js/collapsible-filters/interactions.js

```diff
@@ -214,7 +214,8 @@ export default class Interactions {
     if (!this._disableScrollToTopOnToggle) {
       this.scrollToTop();
     }
-    ANSWERS.components.getActiveComponent('FilterLink').setState({
+    const filterLink = ANSWERS.components.getActiveComponent('FilterLink');
```

Comment:
Is this change related?

Reply:
I was occasionally running into issues with this component not existing (not sure why it would ever not exist, could be related to the map code running before collapsible filters is ready)

Reply:
I'll make it a separate pr